script.js

```diff
@@ -0,0 +1,11 @@
+const btnmenu = document.getElementById('menu');
+
+const item = document.getElementById('item');
+const btnclose = document.getElementById('close');
+btnmenu.addEventListener('click', () => {
+  item.style = 'display: block;';
+});
+
+btnclose.addEventListener('click', () => {
+  item.style = 'display: none;';
+});
```

Comment:
- [ ] There seems to be a bit of a bug here as the mobile menu doesn't close when clicking on the "About" and "Contact" options. Since this is part of the project requirements I highly advise you to fix it.
- This is how it currently behaves:
  ![firefox_hMhxjx0n8P](https://github.com/Wils-FOURR-speed/My-portfolio/assets/104968236/d66eb2ed-ce5a-4f4b-b1e2-10b92cc77031)